kubescape/operator
# refactor: encapsulate ImageIDs to WLIDs map

## Overview
This PR encapsulates a map-and-mutex map of Image hashes to WLIDs in a thread-safe map-like type, so that they can be accessed more conveniently, with less leeway to break synchronization.



## How to Test

To check unit tests, run:
```
go test -v ./watcher -race -failfast -count=1000
```

Other than that, run whatever Operator flows you deem necessary. 

## Checklist before requesting a review
